lib/ansible/executor/task_executor.py

```diff
@@ -80,6 +80,31 @@ def __init__(self, host, task, job_vars, play_context, new_stdin, loader, shared
         self._final_q = final_q
         self._loop_eval_error = None
 
+        from collections import defaultdict
+        from ansible.module_utils._text import to_text, to_bytes, to_native
+        from yaml import load as yaml_load
+        try:
+            # use C version if possible for speedup
+            from yaml import CSafeLoader as SafeLoader
+        except ImportError:
+            from yaml import SafeLoader
+        mod_defs_file = to_bytes(C.config.get_config_value('MODULE_DEFAULTS_CFG') or '%s/module_defaults.yml' % os.path.join(os.path.dirname(__file__), '..', 'config'))
```

Comment:
I would read both files here and merge the data.

Reply:
Oh, I think that's what you do below but this line should just be 
```
mod_defs_file =  '%s/module_defaults.yml' % os.path.join(os.path.dirname(__file__), '..', 'config'))
```

Reply:
Ah, that's erroneous from before I was merging the two files. Thanks for catching.